internal/api/v1/tenant.go

```diff
@@ -0,0 +1,47 @@
+package v1
+
+import (
+	"net/http"
+
+	"github.com/flexprice/flexprice/internal/api/dto"
+	"github.com/flexprice/flexprice/internal/service"
+	"github.com/gin-gonic/gin"
+)
+
+type TenantHandler struct {
+	tenantService *service.TenantService
+}
+
+func NewTenantHandler(tenantService *service.TenantService) *TenantHandler {
+	return &TenantHandler{tenantService: tenantService}
+}
+
+func (h *TenantHandler) CreateTenant(c *gin.Context) {
```

Comment:
Add annotations so that it is available in the API specs refer any api route and see the pattern in comments example `api/v1/wallet.go`